### ISSUE TITLE
Error compilation in janapy

### ISSUE THREAD

**Issue by DraTeots** [JeffersonLab/JANA2#97]
tag 2.0.2

```
[ 65%] Building CXX object src/plugins/janacontrol/src/CMakeFiles/janacontrol_plugin.dir/janacontrol.cc.o
[ 66%] Linking CXX shared library janapy.so
/usr/bin/ld: /usr/local/lib/libpython3.7m.a(abstract.o): relocation R_X86_64_32S against `_Py_NotImplementedStruct' can not be used when making a shared object; recompile with -fPIC
/usr/local/lib/libpython3.7m.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
src/plugins/janapy/CMakeFiles/janapy.dir/build.make:85: recipe for target 'src/plugins/janapy/janapy.so' failed
make[2]: * [src/plugins/janapy/janapy.so] Error 1
CMakeFiles/Makefile2:444: recipe for target 'src/plugins/janapy/CMakeFiles/janapy.dir/all' failed
make[1]: * [src/plugins/janapy/CMakeFiles/janapy.dir/all] Error 2
make[1]: * Waiting for unfinished jobs....
[ 68%] Building CXX object src/plugins/janacontrol/src/CMakeFiles/janacontrol_plugin.dir/JControlZMQ.cc.o
[ 69%] Linking CXX shared library janarate.so
[ 71%] Linking CXX shared library streamDet.so
[ 71%] Built target janarate
[ 73%] Linking CXX shared library janacontrol.so
[ 73%] Built target janacontrol_plugin
[ 73%] Built target streamDet
Makefile:129: recipe for target 'all' failed
make: * [all] Error 2
Execution done. Return code = 2

ERROR
: Command return code != 0. COMMAND CONTENT:
cmake -Wno-dev -DCMAKE_INSTALL_PREFIX=/home/microway/install_escalate/jana/jana-v2.0.2 -DCMAKE_CXX_STANDARD=17   /home/microway/install_escalate/jana/src/v2.0.2&& cmake --build . -- -j 4&& cmake --build . --target install
```

This happens on ubuntu 16.04

**Comment by DraTeots** [comment]
Guess this one could be just close as its old jana version

**Comment by nathanwbrei** [comment]
Don't close it just yet, I want to make sure that my fix for #98 prevents this from popping up again as well. Also, is there are a reason you are using v2.0.2 as opposed to v2.0.3?

**Comment by DraTeots** [comment]
> Also, is there are a reason you are using v2.0.2 as opposed to v2.0.3?

No. Cristiano had not the newest EJPM version where v2.0.2 was set as default (before 2.0.3 came)
